dataminer-overview/Pricing/MACC_and_DataMiner_credits.md

```diff
@@ -4,19 +4,17 @@ uid: MACC_and_DataMiner_credits
 
 # Maximizing your azure investments with MACC and DataMiner credits
 
-DataMiner credits can be conveniently obtained from the following sources:
+([DataMiner credits](xref:Pricing_Usage_based_service#usage-terms)) can be conveniently obtained from the following sources:
 
 - Skyline Communications ([contact us](https://www.skyline.be/contact/))
 - DataMiner Business Partners (see [Partners](https://community.dataminer.services/partners/) on DataMiner Dojo)
 - Microsoft Azure Marketplace ([Microsoft Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/skylinecommunicationsnv1644148291022.dataminer?tab=Overview))
 
-DataMiner credits offer a flexible currency option for subscribing to DataMiner software licenses and hosting services.
-
-When procured through the Microsoft Azure Marketplace, these credits are eligible for DataMiner hosting services, including DataMiner as a Service and Storage as a Service.
+When DataMiner credits are procured through the Microsoft Azure Marketplace, they are eligible for DataMiner Cloud Platform Services, including DataMiner as a Service (DaaS) and Storage as a Service (STaaS) services.
```

Comment:
Could you double-check whether the use of "DataMiner Cloud Platform" here and on line 17 is OK? Some time ago, all references  to "Cloud Platform" were removed from the documentation and we were told to avoid this terminology, so I'm concerned about introducing this again.